src/app/label/[imageKey]/finish/_components/circular-progress.tsx

```
@@ -0,0 +1,67 @@
+  const center = size / 2;
+  const radius = center - strokeWidth / 2;
+  const circumference = 2 * Math.PI * radius;
+
+  const offset = circumference * (1 - progress / 100);
+
+  return (
+    <svg
+      width={size}
+      height={size}
+      className={cn('-rotate-90', className)}
+    >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)